docs/rubrics/v0.1/agentic-delegation.md

```
@@ -0,0 +1,13 @@
+# Rubrica v0.1 — agentic-delegation
+
+> PRÉ-REGISTRADA 2026-08-15 antes de qualquer avaliação (council D5).
+> Critérios derivados de failure modes em traces reais (evidence). Todo critério tem âncora; um anchor pode fundamentar 2 critérios da mesma classe.
+> Formulados como CLASSE de falha, não incidente único.
+
+## C1 — Detecção de não-entrega
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)